--- origsrc/hdf5-1.8.11/configure.ac    2013-05-08 08:34:56.000000000 +0200
+++ src/hdf5-1.8.11/configure.ac        2013-05-18 22:57:11.985530700 +0200
@@ -2188,7 +2188,15 @@ AC_TRY_COMPILE([
 ## ----------------------------------------------------------------------
 ## How do we figure out the width of a tty in characters?
 ##
-AC_CHECK_FUNCS([_getvideoconfig gettextinfo GetConsoleScreenBufferInfo])
+AC_CHECK_FUNCS([_getvideoconfig gettextinfo])
+case "`uname`" in
+  CYGWIN*)
+    ;;
+  *)
+    AC_CHECK_FUNCS([GetConsoleScreenBufferInfo])
+    ;;
+esac
+
 AC_CHECK_FUNCS([_scrsize ioctl])

 AC_MSG_CHECKING([for struct videoconfig])