diff --git a/conf/petscconf.py b/conf/petscconf.py
index a44164d..bde2ecd 100644
--- a/conf/petscconf.py
+++ b/conf/petscconf.py
@@ -12,10 +12,10 @@ __all__ = ['setup',

 # --------------------------------------------------------------------

-from conf.baseconf import PetscConfig as BasePetscConfig
-from conf.baseconf import setup, Extension
-from conf.baseconf import config, build, build_src, build_ext
-from conf.baseconf import test, sdist
+from .baseconf import PetscConfig as BasePetscConfig
+from .baseconf import setup, Extension
+from .baseconf import config, build, build_src, build_ext
+from .baseconf import test, sdist

 # --------------------------------------------------------------------

diff --git a/setup.py b/setup.py
index a4c62f5..eef8009 100755
--- a/setup.py
+++ b/setup.py
@@ -107,9 +107,11 @@ def run_setup():
             metadata['install_requires'] += ["petsc>=3.4,<3.5"]
     #
     setup(packages     = ['petsc4py',
-                          'petsc4py.lib',],
+                          'petsc4py.lib',
+                          'petsc4py.conf'],
           package_dir  = {'petsc4py'     : 'src',
-                          'petsc4py.lib' : 'src/lib'},
+                          'petsc4py.lib' : 'src/lib',
+                          'petsc4py.conf': 'conf'},
           package_data = {'petsc4py'     : ['include/petsc4py/*.h',
                                             'include/petsc4py/*.i',
                                             'include/petsc4py/*.pxd',