diff --git a/conf/petscconf.py b/conf/petscconf.py
index 2ef0a82..bb0828a 100644
--- a/conf/petscconf.py
+++ b/conf/petscconf.py
@@ -12,8 +12,8 @@ __all__ = ['setup',

 # --------------------------------------------------------------------

-from conf.baseconf import setup, Extension
-from conf.baseconf import config, build, build_src, build_ext
-from conf.baseconf import test, sdist
+from .baseconf import setup, Extension
+from .baseconf import config, build, build_src, build_ext
+from .baseconf import test, sdist

 # --------------------------------------------------------------------
diff --git a/setup.py b/setup.py
index 68fb515..c82039c 100755
--- a/setup.py
+++ b/setup.py
@@ -106,9 +106,11 @@ def run_setup():
             metadata['install_requires'] += ["petsc>=3.5,<3.6"]
     #
     setup(packages     = ['petsc4py',
-                          'petsc4py.lib',],
+                          'petsc4py.lib',
+                          'petsc4py.conf'],
           package_dir  = {'petsc4py'     : 'src',
-                          'petsc4py.lib' : 'src/lib'},
+                          'petsc4py.lib' : 'src/lib',
+                          'petsc4py.conf': 'conf'},
           package_data = {'petsc4py'     : ['include/petsc4py/*.h',
                                             'include/petsc4py/*.i',
                                             'include/petsc4py/*.pxd',