diff -ru numpy-1.7.1/numpy/distutils/fcompiler/gnu.py numpy-1.7.1_cray_compilers/numpy/distutils/fcompiler/gnu.py
--- numpy-1.7.1/numpy/distutils/fcompiler/gnu.py	2013-04-07 01:04:05.000000000 -0400
+++ numpy-1.7.1_cray_compilers/numpy/distutils/fcompiler/gnu.py	2013-11-16 00:16:31.000000000 -0500
@@ -33,13 +33,13 @@

     def gnu_version_match(self, version_string):
         """Handle the different versions of GNU fortran compilers"""
-        m = re.match(r'GNU Fortran', version_string)
+        m = re.search(r'GNU Fortran', version_string)
         if not m:
             return None
-        m = re.match(r'GNU Fortran\s+95.*?([0-9-.]+)', version_string)
+        m = re.search(r'GNU Fortran\s+95.*?([0-9-.]+)', version_string)
         if m:
             return ('gfortran', m.group(1))
-        m = re.match(r'GNU Fortran.*?([0-9-.]+)', version_string)
+        m = re.search(r'GNU Fortran.*?([0-9-.]+)', version_string)
         if m:
             v = m.group(1)
             if v.startswith('0') or v.startswith('2') or v.startswith('3'):
