diff -Nru parmetis-4.0.3/metis/GKlib/error.c-orig parmetis-4.0.3/metis/GKlib/error.c
--- parmetis-4.0.3/metis/GKlib/error.c-orig        2016-04-25 22:49:02.000000000 -0500
+++ parmetis-4.0.3/metis/GKlib/error.c     2016-04-25 22:49:11.000000000 -0500
@@ -18,7 +18,7 @@
 
 /* These are the jmp_buf for the graceful exit in case of severe errors.
    Multiple buffers are defined to allow for recursive invokation. */
-#define MAX_JBUFS 128
+#define MAX_JBUFS 24
 __thread int gk_cur_jbufs=-1;
 __thread jmp_buf gk_jbufs[MAX_JBUFS];
 __thread jmp_buf gk_jbuf;
